Update playbooks_error_handling.rst (#2095)
ansible/ansible-documentation: docs/docsite/rst/playbook_guide/playbooks_error_handling.rst

        - '"ERROR" in result.stderr'
        - result.rc == 2

> If you want to introduce your own variables, to avoid repeating a certain term, you can simply reference them in your conditionals

Reviewer: ```suggestion
You can reference simple variables in conditionals to avoid repeating certain terms, as in the following example:
```

.. code-block:: yaml

  - name: Example playbook
    hosts: myHosts
    vars:
       log_path: /home/ansible/logfolder/
       log_file: log.log

   tasks:
      - name: Create empty log file
       ansible.builtin.shell: mkdir {{ log_path }} || touch {{log_path }}{{ log_file }}
       register: tmp
       changed_when:
         - tmp.rc == 0
         - 'tmp.stderr != "mkdir: cannot create directory ‘" ~ log_path ~ "’: File exists"'

.. note::

>    Notice the missing ``{{ }}`` around log_path. 

Reviewer: ```suggestion
   Notice the missing double curly braces ``{{ }}`` around the ``log_path`` variable in the ``changed_when`` statement. 
```

>    Just like ``when`` these two conditionals do not require templating delimiters (``{{ }}``) as they are implied.

Reviewer: ```suggestion
   Just like ``when`` these two conditionals do not require templating delimiters (``{{ }}``) because they are raw Jinja2 expressions.
```

Reviewer: Maybe we could add a link to this section of the docs: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_conditionals.html#basic-conditionals-with-when

>    If you still use them, ansible will raise a warning ``[WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}.``   

Reviewer: I don't think we should include the exact warning text here. Granted this might be helpful to users who search for the warning but it adds maintenance overhead to the docs. I also don't believe it's good practice to document errors and warnings - better to document just the behaviour.

>    If you still use them, ansible will raise a warning ``[WARNING]: conditional statements should not include jinja2 templating delimiters such as {{ }} or {% %}.``   

Reviewer 2: ```suggestion
   If you still use them, ansible will raise a warning  that conditional statements should not include jinja2 templating delimiters.   
```
